### DISCUSSION TITLE
how to delete incorrect peak efforts

### DISCUSSION THREAD

**Discussion by Patzemann** [SauceLLC/sauce4strava#204] [Q&A] [answered]
Hi @all,

how can I delete incorrect peak efforts e.g. because of a powermeter error? Thought I`ve done this already but I don't know how to do it anymore. Thx for help

**Comment by mayfield** [comment, marked as answer]
Hi @Patzemann ,

Select the activity in question in Sauce Performance so it shows up on the right column and then click the Edit icon.  From there you can select the Exclude from Peaks option and save it.

**Reply by Patzemann** [reply]
Great! Thanks a lot.
